doc/releases.md

```diff
@@ -616,13 +623,12 @@ This script will use the promoted builds and changelog to generate the post. Run
   ARMv6 builds. Any downloads that are missing will have `*Coming soon*` next to
   them. It's your responsibility to manually update these later when you have
   the outstanding builds.
-* The SHASUMS256.txt.asc content is at the bottom of the post. When you update
+* The `SHASUMS256.txt.asc` content is at the bottom of the post. When you update
   the list of tarballs you'll need to copy/paste the new contents of this file
   to reflect those changes.
-* Always use pull-requests on the nodejs.org repo. Be respectful of that working
-  group, but you shouldn't have to wait for PR sign-off. Opening a PR and
-  merging it immediately _should_ be fine. However, please follow the following
-  commit message format:
+* Always use pull-requests on the `nodejs/nodejs.org` repository. Be respectful
+  of that working group, but you do not have to wait for PR sign-off. Please
```

Comment:
```suggestion
  of that team, but you do not have to wait for PR sign-off. Please
```


`nodejs/nodejs.org` is no longer a WG: https://github.com/nodejs/TSC/pull/761

Reply:
Updated